src/Model/MovingModel.py

```diff
@@ -76,7 +80,16 @@ def create_moving_model():
 
     moving_dict_container.set("dict_windowing_moving", dict_windowing)
 
-    pixel_values = convert_raw_data(dataset)
+    slope = 1
+    intercept = 0
+
+    if not moving_dict_container.has_attribute("scaled"):
```

Comment:
Don't quite understand this block of code here. The if/else does nothing as `pixel_values` is set again right under this block, meaning anything in the if/else is discarded and unused.

Also, `is_pt` is the rescale intercept, right? Why hard code this value instead of grabbing it from the DICOM file? And then if this is true, why is the rescale intercept stored in both `is_pt` and `intercept` but with potentially different values?

Also a little confused about the rescale slope, which is specified as the integer 1 but passed into `convert_raw_data` as a boolean.

The name of `is_pt` is a bit misleading too, as it implies this would be a boolean when it is in fact an integer.

The code all works fine, I just think my confusion comes down to the variable names.

Reply:
I have the same confusion with is_pt

I think it's really important to distinguish between scaling of the underlying data and rescaling for presentation.

Reply:
> Also, is_pt is the rescale intercept, right? Why hard code this value instead of grabbing it from the DICOM file? And then if this is true, why is the rescale intercept stored in both is_pt and intercept but with potentially different values?

This is a bit confusing so I will also clarify and use examples: 
- in [DICOM-RT-02](https://onkodicom.com.au/sites/default/files/2020-01/DICOM-RT-02.zip) the scans have a rescale slope of 1 and rescale intercept of -1024, however the image itself is already bright enough for Andrew, so when applying the rescale, I add 1024 so 1024-1024=0 i.e. the intercept does not change. 
- in [LUNG1-010](https://redmine.didymodesigns.com.au/attachments/762) we have a rescale slope of 1 and an intercept of 0, meaning this data is already in Hounsfield, however [Andrew's comment](https://onkoworkspace.slack.com/archives/CGRSJ6H16/p1631838677111500) means this is too dark. Testing i found increasing the intercept by 1024 to be the best way to do it. Another way is that DICOM files store the maximum bit depth, which does work, however
- in [test image fusion dataset](https://redmine.didymodesigns.com.au/attachments/744), the slope is 1, and the rescale is 0, but almost all the data is greater than -1024, except for the endges which have a value of -2048, the maximum bit depth here because of this is -2048 and using this to adjust the intercept will make every other value brighter than recorded. This does mean there will be values less than 0 but that is due to the data coming in

So the reason there is an is_pt is because there is no good way in DICOM to rescale out of Hounsfield, especially if the image is already in hounsfield.

I also just realised this is meant to be for CT, not PT, as PT does not need this intercept change, so I'll fix this

Reply:
All this data is accessablie in thr Dicom Tree tab

Reply:
> Don't quite understand this block of code here. The if/else does nothing as pixel_values is set again right under this block, meaning anything in the if/else is discarded and unused.

This was a mistake and has been fixed, thanks for the pickup.

Reply:
I think I understand now. Thank you for the explanation, it was very helpful!